os_stub/cryptlib_openssl/pk/x509.c

```diff
@@ -2156,3 +2161,358 @@ bool libspdm_x509_get_cert_from_cert_chain(const uint8_t *cert_chain,
 
     return false;
 }
+
+bool libspdm_set_subject_name(X509_NAME *x509_name, char *subject_name)
+{
+    int ret;
+    uint8_t index;
+    char *char_start;
+    char *char_end;
+    char temp[MAX_SBUJECT_NAME_LEN];
+
+    ret = 0;
+    char_start = NULL;
+    char_end = NULL;
+
+    /* X.509 DN attributes from RFC 5280, Appendix A.1. */
+    char *standard_subject[] = {
+        "CN=", "commonName=", "C=", "countryName=", "O=", "organizationName=","L=", "locality=",
+        "R=", "OU=", "organizationalUnitName=", "ST=", "stateOrProvinceName=", "emailAddress=",
+        "serialNumber=", "postalAddress=", "postalCode=", "dnQualifier=", "title=","surName=",
+        "SN=","givenName=","GN=", "initials=", "pseudonym=", "generationQualifier=",
+        "domainComponent=", "DC="
+    };
+
+    char *subject_set[] = {
+        "CN", "commonName", "C", "countryName", "O", "organizationName","L", "locality","R", "OU",
+        "organizationalUnitName", "ST", "stateOrProvinceName", "emailAddress", "serialNumber",
+        "postalAddress", "postalCode", "dnQualifier", "title","surName", "SN","givenName","GN",
+        "initials", "pseudonym", "generationQualifier", "domainComponent", "DC"
+    };
```

Comment:
subject_set is similar to standard_subject.
Please remove one of them.

Reply:
Yes. I have deleted one of them.